### ISSUE TITLE
Low serum osmolality

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9930]
**Preferred term label:**
Low serum osmolality

**Synonyms**
Low serum osmolarity

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
SIADH

**Your nano-attribution (ORCID)**

**Comment by MickeySegal** [comment]
There should also be High serum osmolality/osmolarity, as in hyperosmolar hyperglycemic syndrome

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000680
- label: Low serum osmolality
- parents: [Abnormality of fluid regulation (HP:0011032)](https://hpo.jax.org/app/browse/term/HP:0011032)
- pmid: PMID:35779183
- def: Serum osmolality, which is defined as the sum of the osmolalities of every single dissolved particle in the blood such as sodium and associated anions, potassium, glucose, and urea, is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9930
- synonyms: 


Adding new term: 
- ID: HP:6000681
- label: High serum osmolality
- parents: [Abnormality of fluid regulation (HP:0011032)](https://hpo.jax.org/app/browse/term/HP:0011032)
- pmid: PMID:35779183
- def: Serum osmolality, which is defined as the sum of the osmolalities of every single dissolved particle in the blood such as sodium and associated anions, potassium, glucose, and urea, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9930
- synonyms:
